nomad/local/nomad_run_local_infra.sh

```diff
@@ -7,7 +7,7 @@ echo "Deploying local infrastructure."
 
 # Wait a short period of time before attempting to deploy infrastructure
 # shellcheck disable=SC2016
-timeout 30 bash -c -- 'while [[ -z $(nomad status 2>&1 | grep running) ]]; do printf "Waiting for nomad-agent\n";sleep 1;done'
+timeout 120 bash -c -- 'while [[ -z $(nomad status 2>&1 | grep running) ]]; do printf "Waiting for nomad-agent\n";sleep 1;done'
```

Comment:
We were timing out before nomad had started

Reply:
120 feels long to me, but, eh